File: Atividades Udemy/Java/secao06/atividade05.java
package secao06;

import java.util.Scanner;

public class atividade05 {

	public static void main(String[] args) {
		float p, m;
		String e = "excesso";
		Scanner teclado = new Scanner(System.in);
		
		System.out.println("Informe o peso de peixes: ");
		p = teclado.nextFloat();
		
		if(p > 50) {
			m = (float)(p - 50) * (float)4.00;
			System.out.printf("Você deverá pagar R$ %.2f em multas", m);
		} else {
			m = 0;
			e = "0.0";
			System.out.println("Multas: " + m);
			System.out.println("Excesso: " + e);
		}
		teclado.close();
	}

}

File: Atividades Udemy/Python/secao06/atividade05.py
e = 0
m = 0
peso = float(input("Informe o peso dos peixes: "))

if (peso > 50):
    e = (peso - 50)
    m = e * 4
    print(f"Peso: {peso:.2f} kg")
    print(f"Excesso: {e:.2f} kg")
    print(f"Multa: R$ {m:.2f}")
else:
    print("Sem excesso.")